CHANGELOG.md

```diff
@@ -1,55 +1,12 @@
 # Changelog
-All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
-This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+This project adheres to [Semantic
+Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+All notable changes to this project will be documented at
+<https://github.com/packethost/packngo/releases>. Drafts release notes may be
+used to track features that will be available in future releases.
```

Comment:
Including changelogs as files in the git repo, like manually updated version stamps, creates an extra hurdle for automated releases.

In many projects, the Github releases page acts a one stop shop for artifacts and release notes.

Is this the only copy of the release notes? The release notes can often mirror the tag comment. The tag comment can mirror the changelog entry.  The changelog entry could mirror the git commit description that bumped the tag.  There are too many copies of the same information.

The most flexible of these is the Github release notes, which can be modified through the UI and can be fetched/subscribed to as ATOM.  Goreleaser (or [release drafter](https://github.com/marketplace/actions/release-drafter)) can manage the release notes.

It's also worth noting that the CHANGELOG hasn't been kept up to date. Moving to a process like this takes away the manual (forgotten) step and simplifies the practices that we need to apply/remember.

In packngo, we have to update the version stamp today on release, but we could move that work to a robot commit.

If we move to a standardized git commit format, we can have robots verify those. If we use a Github Workflow button to trigger tag releases, we can verify that the version.go file matches the version chosen in the UI (or that version.go contains a unique tag, or we can let a Github action commit the version.go change).

I'm all for automating these small details.

Reply:
> non-normal
I've been seeing this more: https://www.google.com/search?q=move+changelog+to+github+releases (several examples of Github issues where projects are doing this, notably jenkins)

Reply:
> It's also worth noting that the CHANGELOG hasn't been kept up to date. Moving to a process like this takes away the manual (forgotten) step and simplifies the practices that we need to apply/remember.

Yep its definitely been a pain in the past, also considering merge conflicts too. 

> In packngo, we have to update the version stamp today on release, but we could move that work to a robot commit.

It would be wonderful for the robot commit to also update CHANGELOG.md

> If we move to a standardized git commit format, we can have robots verify those. If we use a Github Workflow button to trigger tag releases, we can verify that the version.go file matches the version chosen in the UI (or that version.go contains a unique tag, or we can let a Github action commit the version.go change).
> 
> I'm all for automating these small details.

Sounds good to me.

Reply:
On another note, one of the links I followed from your goole search link kept the historic entries and added a something along the lines of "new changelogs will be in GitHub Release $url". Why not do that here too? Or do you plan to replicate the changelog over to the releases?

Reply:
I'd rather relegate the changelog to https://github.com/packethost/packngo/releases, where the past releases / CHANGELOG entries are already recorded.